src/TTextEdit.cpp

```diff
@@ -225,6 +226,7 @@ void TTextEdit::initDefaultSettings()
 void TTextEdit::updateScreenView()
 {
     if (isHidden()) {
+        mFontMetrics = QFontMetrics(mpHost->mDisplayFont);
```

Comment:
Why are you using `mpHost->mDisplayFont` and not `mDisplayFont` here?

And when you do use the latter the following lines can access the `mFontMetrics` member instead of each of them each retrieving a fresh instance of the relevant `QFontMetrics` as well...! :wink:

Reply:
Sorry my bad, cut&paste misprint. I'll fix it + I was trying to minimize the modified code. I'm fixing it.